app/views/main.scala.html

```diff
@@ -27,42 +35,13 @@
       media="screen"
       href="/assets/bundle/vendors~main.css?nocache=@(webknossos.BuildInfo.commitHash)"
     />
-    <script>
-      let initialTheme = "@(selectedTheme)";
-      if (initialTheme === "auto") {
-        initialTheme =
-          window.matchMedia("(prefers-color-scheme: dark)").media !== "not all" &&
-          window.matchMedia("(prefers-color-scheme: dark)").matches
-            ? "dark"
-            : "light";
-      }
-      document.documentElement.style.display = "none";
-      document.head.insertAdjacentHTML(
-        "beforeend",
-        "<link " +
-          'id="primary-stylesheet" ' +
-          'rel="stylesheet" ' +
-          'type="text/css" ' +
-          'media="screen" ' +
-          'href="/assets/bundle/' +
-          initialTheme +
-          '.css?nocache=@(webknossos.BuildInfo.commitHash)" ' +
-          "onload=\"document.documentElement.style.display = ''\" " +
-          "/>",
-      );
-    </script>
```

Comment:
This script was moved into `main.tsx` and the `selectedTheme` and the commit hash are accessed via the meta tags